deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -519,6 +520,10 @@ def __init__(self,
         self._enable_universal_checkpoint()
         self._param_slice_mappings = self._create_param_mapping()
 
+    def destroy(self):
```

Comment:
This is needed in the other optimizer implementations: 
1. https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/runtime/bf16_optimizer.py
2. https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/runtime/fp16/fused_optimizer.py
3. https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/runtime/fp16/unfused_optimizer.py